server/views/partials/plan-header/plan-header-mini.njk

```diff
@@ -17,6 +17,9 @@
                     <dt>{{ locale.common.planHeader.mini.dateOfBirthLabel }}</dt>
                     <dd>{{ data.popData.dateOfBirth }} </dd>
                 </div>
+                <div>
```

Comment:
we don't need this yet, and it doesn't render properly on all pages, let's take this out.